content/hi/docs/reference/glossary/customresourcedefinition.md

```diff
@@ -0,0 +1,19 @@
+---
+title: कस्टम रिसोर्स डिफिनिशन
```

Comment:
@CloudShiva7cr  To maintain the consistent look of the page provide the english term in bracket as well.
`कस्टम रिसोर्स डिफिनिशन (Custom Resource Defination)`

Reply:
Sure. Add english title bracket (Custom Resource Definition) in it. 
thanks!!